src/mpid/ch4/src/ch4_init.h

```diff
@@ -248,8 +241,11 @@ MPL_STATIC_INLINE_PREFIX int MPID_Init(int *argc,
     }
 #endif
 
+    mpi_errno = MPIDI_NM_mpi_init_hook(rank, size, appnum, &MPIR_Process.attrs.tag_ub,
```

Comment:
Got it. This will actually be useful for some `MPI_Init` optimizations we discussed, too.